docs/macaroons.md

```diff
@@ -81,7 +81,14 @@ methods. This means a few important things:
 You can also run `lnd` with the `--no-macaroons` option, which skips the
 creation of the macaroon files and all macaroon checks within the RPC server.
 This means you can still pass a macaroon to the RPC server with a client, but
-it won't be checked for validity.
+it won't be checked for validity. Note that disabling authentication of a server
+that's listening on a public interface is not allowed. This means the
+`--no-macaroons` option is only permitted when the RPC server is in a private
+network. In CIDR notation, the following IPs are considered private,
+- [`169.254.0.0/16` and `fe80::/10`](https://en.wikipedia.org/wiki/Link-local_address).
```

Comment:
Note that these lines will be longer than 80 before rendering. However, after rendering, they should be fine.

Reply:
Yeah, in markdown it's hard to adhere to the character limit because of links. That's totally fine.